src/Xamarin.Android.Build.Tasks/ILRepack.targets

```diff
@@ -1,7 +1,13 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
 <Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 	<UsingTask AssemblyFile="$(MSBuildThisFileDirectory)..\..\packages\ILRepack.Lib.MSBuild.Task.2.0.15.4\build\ILRepack.Lib.MSBuild.Task.dll" TaskName="ILRepack" />
-	<Target Name="ILRepacker" AfterTargets="Build">
+	<PropertyGroup>
+		<_ILRepacker_stamp>$(IntermediateOutputPath)ILRepacker.stamp</_ILRepacker_stamp>
+	</PropertyGroup>
+	<Target Name="ILRepacker"
+		AfterTargets="Build"
```

Comment:
`AfterTargets="Build"` can be dangerous, because of a bug in MSBuild: Microsoft/msbuild#3345.

Can you instead do `AfterTargets="AfterBuild"`? Confusingly, `AfterBuild` completes _before_ `Build`.

Reply:
Thanks for the tip! Updated :)